gems/aws-sdk-core/lib/aws-sdk-core.rb

```diff
@@ -9,60 +9,70 @@ module AWS
   module SDK
     # Namespace for AWS::SDK Core components
     module Core
-      # @return true if v4 aws-sdk-sts is available
-      def self.sts_loaded?
-        if @use_sts.nil?
-          @use_sts =
-            begin
-              require 'aws-sdk-sts'
-              true
-            rescue LoadError, NameError
-              false
-            end
+      class << self
+        # @return [Hash] Returns a hash of default configuration options shared
+        #   by all constructed clients.
+        def config
```

Comment:
Can we do this as a separate PR, it feels un-related to the repl and is a pretty big change.

Reply:
yeah